app/connectors/FinancialDetailsConnector.scala

```diff
@@ -230,11 +226,13 @@ class FinancialDetailsConnector @Inject()(
       }
   }
 
-  def getOutstandingCharges(idType: String, idNumber: String, taxYear: String)
-                           (implicit headerCarrier: HeaderCarrier): Future[OutstandingChargesResponseModel] = {
+  def getPayments(taxYear: Int)
+                 (implicit headerCarrier: HeaderCarrier, mtdUser: MtdItUser[_]): Future[PaymentsResponse] = {
 
+    val dateFrom: String = s"${taxYear - 1}-04-06"
```

Comment:
:wrench: - if we can use taxYear with TaxYear type, as TaxYear provides required conversion to YYYY-MM-DD format;